SocialCareCaseViewerApi/V1/UseCase/Interfaces/IWarningNoteUseCase.cs

```diff
@@ -1,13 +1,11 @@
-using System.Collections.Generic;
 using SocialCareCaseViewerApi.V1.Boundary.Requests;
 using SocialCareCaseViewerApi.V1.Boundary.Response;
-using SocialCareCaseViewerApi.V1.Domain;
 
 namespace SocialCareCaseViewerApi.V1.UseCase.Interfaces
 {
     public interface IWarningNoteUseCase
     {
         PostWarningNoteResponse ExecutePost(PostWarningNoteRequest request);
-        List<WarningNote> ExecuteGet(GetWarningNoteRequest request);
+        ListWarningNotesResponse ExecuteGet(long personId);
```

Comment:
If you agree with SocialCareCareCaseViewerApiController comment this could be reverted.